### ISSUE TITLE
librispeech SPOKEN_NOISE vs UNK

### ISSUE THREAD

**Issue by danpovey** [kaldi-asr/kaldi#632]
In Librispeech, we have <SPOKEN_NOISE> in oov.txt instead of <UNK>.  It would be better to have <UNK> in oov.txt and no <SPOKEN_NOISE>, because some later script (well, the --remove-oov option to mkgraph) assume that the unk symbol in the LM has the same word-id as this.
Dan

**Comment by vdp** [comment]
I can't seem to remember how exactly we are using SPOKEN_NOISE and UNK in the recipe.. Is this just a question of replacing SPOKEN_NOISE w/ UNK in lines 55 and 211 of run.sh? Also, I can't see the "remove-oov" option in the current version of utils/mkgraph.sh- perhaps this is related to the changes that were necessary for the chain paper?

**Comment by vdp** [comment]
Nevermind about the "remove-oov" option- I see it's in the "chain" branch.

**Comment by danpovey** [comment]
Yes I think it is as simple as that.  They have the same pronunciation so
it won't make any difference in general.  Would you mind doing a PR, to the
chain branch?
Dan
